index.js

```diff
@@ -2,7 +2,8 @@
 const aug = require('aug');
 const url = require('url');
 const pluginDefaults = {
-  logTags: ['content-security-policy-report'],
+  httpsOnly: false, // only respond if request is https
```

Comment:
is there any way to add a test for this?

Reply:
Added a test for this by just manually setting server.info.protocol before server.inject.